eslint.config.mjs

```diff
@@ -74,6 +78,17 @@ export default [
           argsIgnorePattern: '^_',
         },
       ],
+
+      '@typescript-eslint/restrict-template-expressions': [
```

Comment:
According to https://typescript-eslint.io/rules/restrict-template-expressions/ this is part of `plugin:@typescript-eslint/recommended-type-checked`. Would it be much work, to use the more general plugin?

Reply:
Both the `defaultOptionsRecommended` and `defaultOptionsStrict` are not suited for us.
* recommended: allows a lot of crazy stuff
* strict: doesn't allow numbers, which we currently accept